qa/pom.xml

```diff
@@ -348,6 +348,17 @@
           <artifactId>openjdk-orb</artifactId>
           <version>${version.org.jboss.openjdk-orb}</version>
         </dependency>
+        <dependency>
+          <groupId>com.sun.corba</groupId>
+          <artifactId>glassfish-corba-omgapi</artifactId>
```

Comment:
Please can you say in a comment in the code what caused you to decide to add this particular artifact (same for the next added dependency)?

Reply:
For `orb` type `idlj` using `glassfish-corba-omgapi` maven dependency as glassfish provides the implementation for `idlj` https://mvnrepository.com/artifact/com.sun.corba